olp-cpp-sdk-authentication/include/olp/authentication/AppleSignInProperties.h

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright (C) 2021 HERE Europe B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ * License-Filename: LICENSE
+ */
+
+#pragma once
+
+#include "AuthenticationApi.h"
+
+namespace olp {
+namespace authentication {
+
+/**
+ * @brief The Apple sign-in properties.
+ */
+class AUTHENTICATION_API AppleSignInProperties {
+ public:
+  /**
+   * @brief Gets the bearer in the authorization request header.
+   *
+   * @return The access token.
+   */
+  const std::string& GetAccessToken() const { return access_token_; }
+
+  /**
+   * @brief Sets the bearer in the authorization request header.
+   *
+   * @note The 'id_token' received after authorization should be used here.
```

Comment:
Is this correct? No mention that this is an Apple id_token taken from Apple authentication. Please adapt this and the above GetAccessToken() Doxygen to mention that we are talking about a Apple token here.